### DISCUSSION TITLE
AWS Modules and TF Versions

### DISCUSSION THREAD

**Discussion by dlewis152** [gruntwork-io/knowledge-base#561] [Help] [answered]
Is there a list anywhere what versions of AWS modules support older version of Terraform? For example https://github.com/gruntwork-io/terraform-aws-security 

Currently I was going to go through them find the change commit https://github.com/gruntwork-io/terraform-aws-security/blame/1c6571ced9e3b9b7740c00846c1cf4526ca1fd8b/modules/cloudtrail/main.tf where it's not pinned to >`1.0` and then find the relevant release tag but was hoping there is a better way.

https://terragrunt.gruntwork.io/docs/getting-started/supported-terraform-versions/

---

<ins datetime="2022-09-19T19:36:02Z">
  <p><a href="https://support.gruntwork.io/hc/requests/109264">Tracked in ticket #109264</a></p>
</ins>

**Comment by rhoboat** [comment, marked as answer]
You can look at the documentation which lists Terraform version compatibilities across the repositories. These are on the docs.gruntwork.io website, under [Guides](https://docs.gruntwork.io/guides/) > [Update Guides](https://docs.gruntwork.io/guides/stay-up-to-date/) > Select any Terraform version listed. There is a table in the deployment walkthrough for every Terraform version listed.

Here's the table for [Terraform 1.1 support](https://docs.gruntwork.io/guides/stay-up-to-date/terraform/terraform-1.1/deployment-walkthrough/step-2-update-references-to-the-gruntwork-infrastructure-as-code-library#version-compatibility-table).
Here it is for [Terraform 0.12](https://docs.gruntwork.io/guides/stay-up-to-date/terraform/terraform-12/version-compatibility-table).

**Reply by dlewis152** [reply]
That seems to only show the minimum version that supports the TF version, and not the maximum.
